YvesIraguha/CodeLab-ReactNative
# #166258383 Home page Screen

#### What does this PR do?
- Build the home screen
#### Description of Task to be completed?
- Create unit tests for home screen component
- Create home screen 
#### How should this be manually tested?
- Clone the repo and check out `ft-home-page-166258383`
- Install all dependencies by `yarn`
- In project root directory run `yarn run test` to run tests. Tests should pass. 
- Run `yarn run start` to start the server 
- Follow the directions given, in the terminal. 
- You should see the list of engineers either in your simulator or your device.
#### Any background context you want to provide?
- The design of the original mockup had to be modified so that the UI is improved on the actual device. 
#### What are the relevant pivotal tracker stories?
# 166258383
#### Screenshots (if appropriate)
- First design 
<img width="538" alt="Screen Shot 2019-06-03 at 11 04 29" src="https://user-images.githubusercontent.com/34691070/58789888-7ea60c00-85ef-11e9-9b42-febfb1515437.png">
- Updated version 
<img width="372" alt="Screen Shot 2019-06-03 at 11 01 24" src="https://user-images.githubusercontent.com/34691070/58789944-98475380-85ef-11e9-9ae3-b9cca28add82.png">
